modules/oxide-icons-default/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/oxide-icons-default",
-  "version": "1.3.3",
+  "version": "1.3.4",
```

Comment:
You should never manually bump versions in the monorepo, as that needs to be done by `lerna` which we do at release time (see the contributing docs for more details). Sorry I know this is inconsistent across the various things.

If it helps, here are the general rules to follow:
- Is it a patch version bump? then don't manually bump the version
- Is it a minor/major version bump? then manually bump the version, except in monorepos

- [ ] Revert manual package.json bump